core/Cargo.toml

```
@@ -26,6 +26,9 @@ serde = { version = "1.0.125", features = ["derive"] }
 serde_json = { version = "1.0.64", features = ["preserve_order"] }
 serde_v8 = { version = "0.8.0" }
 url = { version = "2.2.1", features = ["serde"] }
+rustls = { version = "0.19.0", features = ["dangerous_configuration"] }
```

[reference](https://github.com/denoland/deno/pull/11324/files#r666687437)

```suggestion
rustls = { version = "0.19.1", features = ["dangerous_configuration"] }
```